seqr/utils/elasticsearch/es_search.py

```diff
@@ -208,7 +208,10 @@ def filter_by_annotation_and_genotype(self, inheritance, quality_filter=None, an
             for index in self._indices:
                 family_samples_by_id = self.samples_by_family_index[index]
                 affected_status = _get_family_affected_status(family_samples_by_id, inheritance_filter)
-                self._family_individual_affected_status.update(affected_status)
+                for family_guid, family_affected_status in affected_status.items():
+                    if family_guid not in self._family_individual_affected_status:
+                        self._family_individual_affected_status[family_guid] = {}
+                    self._family_individual_affected_status[family_guid].update(family_affected_status)
```

Comment:
this is the actual bug fix. If index_1 has `affected_status` of `{'F2': {'4': 'A', '5': 'U'}}` and index_2 has `affected_status` of `{'F2': {'5': 'U', '6': 'U'}}` than under the old code `self._family_individual_affected_status['F2']` would equal either `{'4': 'A', '5': 'U'}` or `{'5': 'U', '6': 'U'}` depending on the sorted index order. Under this new code, and what we actually want, it becomes `{'4': 'A', '5': 'U', '6': 'U'}` and we stop getting key errors

Reply:
It is exactly what I mean for "combining individuals" in my previous comment.